docs/pbc/all/search/202212.0/third-party-integrations/integrate-algolia.md

```diff
@@ -1,44 +1,636 @@
-2. Click **Algolia**. This takes you the Algolia app details page.
-3. In the top right corner of the Algolia app details page, click **Connect app**. The message saying that application connection pending is displayed.
-4. Log in to the [Algolia website](https://www.algolia.com).
-5. On the Algolia website, go to **Settings -> Team and Access -> API keys**:
-![algolia-keys](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/search/algolia/integrate-algolia/algolia-keys.png)
-6. Copy the following keys:
-    - Application ID
-    - Search-Only API Key
-    - Admin API Key
-7. Go back to your store's Back Office, to the Algolia app details page.
-8. In the top right corner of the Algolia app details page, click **Configure**.
-9. In the **Configure** pane, populate the _APPLICATION ID_, _SEARCH-ONLY API KEY_, and _ADMIN API KEY_ fields with the values from step 6.
-![algolia-settings](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/search/algolia/integrate-algolia/algolia-settings.png)
-10. Click **Save**.
+To start integration, the Algolia app requires the following Spryker modules:
 
-<figure class="video_container">
-    <video width="100%" height="auto" controls>
-    <source src="https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/search/algolia/integrate-algolia/algolia-integration.mp4" type="video/mp4">
-  </video>
-</figure>
+* `spryker/catalog: "^5.8.0"`
+* `spryker/catalog-extension: "^1.0.0"`
+* `spryker/catalog-price-product-connector: "^1.4.0"`
+* `spryker/category: "^5.11.0"`
+* `spryker/category-storage: "^2.5.0"`
+* `spryker/message-broker-aws: "^1.3.1"`
+* `spryker/price-product: "^4.37.0"`
+* `spryker/product: "^6.29.0"`
+* `spryker/product-approval: "^1.1.0"`
+* `spryker/product-category: "^4.19.0"`
+* `spryker/product-extension: "^1.4.0"`
+* `spryker/product-image: "^3.13.0"`
+* `spryker/product-label "^3.5.0"`
+* `spryker/product-label-storage "^2.6.0"`
+* `spryker/product-review: "^2.9.0"`
+* `spryker/search: "^8.19.3"`
+* `spryker/search-http: "^1.0.0"`
+* `spryker/store: "^1.17.0"`
+* `spryker/merchant-product-offer: "^1.5.0"` (Marketplace only)
+* `spryker/merchant-product-offer-data-import: "^1.1.0"` (Marketplace only)
+* `spryker/merchant-product-offer-search: "^1.4.0"` (Marketplace only)
+* `spryker/price-product-offer-data-import: "^0.7.1"` (Marketplace only)
+* `spryker/product-offer: "^1.4.0"` (Marketplace only)
 
-The Algolia app is now integrated to your store and starts exporting your product data automatically. Wait for a few minutes and go back to the Algolia website, to the **Overview -> Search -> Index** page. 
 
-{% info_block infoBox "Info" %}
+## Integrate Algolia
 
-The more products you have, the longer you have to wait until Algolia finishes the export. The average export speed is around 100 products per minute.
+Algolia integration is not a single step. Follow the steps in the sections below to integrate Algolia.
 
-{% endinfo_block %}
+### Shared configs
```

Comment:
Fix the title. 
The step must start with a verb, in imperative mood—for example:
```suggestion
### Configure shared configs
```